CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.0.23
+
+- Support mdx file
```

Comment:
I don't think this is an accurate description of what's happening, something like "Integrate style-guide documentation into shared component and enable MDX support" would be more appropriate

Reply:
agree and where do we have the storybook version upgrade?